Telerik.JustMock/Core/MockingUtil.CodeGen.cs

```diff
@@ -173,5 +189,34 @@ public static MethodInfo CreateDynamicMethodWithVisibilityChecks(Type returnType
 			var attr = (DispIdAttribute)Attribute.GetCustomAttribute(member, typeof(DispIdAttribute));
 			return attr != null ? (int?)attr.Value : null;
 		}
+
+		public static ProfilerInterceptor.RefReturn<TReturn> CreateDynamicMethodInvoker<TReturn>(object @object, MethodInfo method, object[] args)
```

Comment:
Using keywords for variable names does not provide additional context to the variable. Could you change them to have more meaningful names?

Reply:
@mihail-vladov fixed with [41d482f](https://github.com/telerik/JustMockLite/pull/42/commits/41d482f390d2ad05ddea272d25c4b14dca4e999b)